releasenotes/notes/tls-inbound-all-protocols.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: security
+releaseNotes:
+- |
+  **Fixed** an issue the TLS minimum version to be set to v1.2 only on HTTP ports. This option is now applied to all ports.
```

Comment:
is this controlled by an API option? From the code it seems like we are enforcing anyway. if there's an option, mention the API field?

Reply:
its an env var. we will likely add first class API in the future